website/static/js/tests/projectSettings.test.js

```diff
@@ -0,0 +1,139 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+var assert = require('chai').assert;
+var utils = require('tests/utils');
+var faker = require('faker');
+
+var $osf = require('js/osfHelpers');
+var Raven = require('raven-js');
+
+/* 
+ * Dear sloria,
+ *
+ * I'm sorry for injecting globals. Please forgive me.
```

Comment:
Dear @samchrisinger ,

It's not you, it's me. It was I who made you commit this when I wrote the `projectSettings.js` code long ago.

Sorry for the bad design,

sloria